project.clj

```diff
@@ -9,6 +9,8 @@
 
   :pedantic? :abort
 
+  :exclusions [org.clojure/clojure]
```

Comment:
Interesting.  I'm guessing this is needed because either clj-semver or versioneer is stuck on an old version of Clojure?